src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/HttpRequestHdr.java

```diff
@@ -164,7 +164,7 @@ public byte[] parse(InputStream in) throws IOException {
                     inHeaders = false;
                     firstLine = false; // cannot be first line either
                 }
-                final String reqLine = line.toString();
+                final String reqLine = line.toString(Charset.defaultCharset().name());
```

Comment:
Only ASCII is should be in the headers (see `CharUtils.isAscii` above), so this encoding should be like `StandardCharsets.ISO_8859_1`